packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -0,0 +1,729 @@
+{{page-title "Modal Component"}}
+
+<h2 class="dummy-h2">Modal</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Modals are used to request information or feedback from the user, confirm a decision, or
+    less frequently to provide additional context about a function or feature.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-modal">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-modal">
+    <dt>
+      size
+      <code>enum</code>
+    </dt>
+    <dd>
+      <p>
+        Sets the width of the modal.
+      </p>
+
+      <p>Acceptable values:</p>
+      <ol>
+        <li>small</li>
+        <li class="default">medium</li>
+        <li>large</li>
+      </ol>
+    </dd>
+
+    <dt>color <code>enum</code></dt>
+    <dd>
+      <p>
+        Sets the color scheme for the modal header elements: icon, tagline and title.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+    <dt>onOpen <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is opened.
+      </p>
+    </dd>
+    <dt>onClose <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is closed.
+      </p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+
+  <h4 class="dummy-h4">Contextual components</h4>
+  <p class="dummy-paragraph" id="contextual-components-modal">The title, the content of the modal dialog, and the
+    actions are passed into the modal as yielded components, using the
+    <code class="dummy-code">Header</code>,
+    <code class="dummy-code">Body</code>,
+    <code class="dummy-code">Footer</code>
+    keys.</p>
+  <dl class="dummy-component-props" aria-labelledby="contextual-components-modal">
+    <dt>&lt;[M].Header&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content as the title of the modal dialog.</p>
+      <p>It supports
+        <code class="dummy-code">...attributes</code>
+      </p>
```

Comment:
You don't need this, you're already making it explicit at lines 92-95.
```suggestion
```

Reply:
Indeed, it's redundant – I removed it